karate-core/src/test/java/com/intuit/karate/FileUtilsTest.java

```diff
@@ -24,6 +47,117 @@
 
     private static final Logger logger = LoggerFactory.getLogger(FileUtilsTest.class);
 
+    @Test
+    public void testIsClassPath() {
+        assertFalse(FileUtils.isClassPath("foo/bar/baz"));
+
+        assertTrue(FileUtils.isClassPath("classpath:foo/bar/baz"));
+    }
+
+    @Test
+    public void testIsFilePath() {
+        assertFalse(FileUtils.isFilePath("foo/bar/baz"));
+
```

Comment:
I prefer not having these extra line feeds for unit tests

Reply:
Removed.